homeassistant/components/sensor/recorder.py

```diff
@@ -56,10 +57,12 @@
         DEVICE_CLASS_GAS: {"sum"},
         DEVICE_CLASS_MONETARY: {"sum"},
     },
+    STATE_CLASS_TOTAL: {},
     STATE_CLASS_TOTAL_INCREASING: {},
 }
 DEFAULT_STATISTICS = {
     STATE_CLASS_MEASUREMENT: {"mean", "min", "max"},
+    STATE_CLASS_TOTAL: {"sum"},
```

Comment:
So is the idea for now to just keep track of `sum` and not do the `increasing`/`decreasing` part yet ?

Reply:
Yeah, I have an implementation of it ready, but I think it makes sense to add that in a follow-up PR.